### DISCUSSION TITLE
Pollinations IRL Gallery

### DISCUSSION THREAD

**Discussion by gokaykucuk** [pollinations/pollinations#80] [Pollinations]
Just an idea, at some point, we can get a collection of really good results from pollinations and show it in a gallery in real life. I'm thinking of having some monitors and displaying them on repeat. What do others think about this?

**Comment by jacksongoode** [comment]
Cool idea! Or perhaps some digital display that shows the latest artwork generated. I was also thinking, to add to this thought, that it would make sense to create a live, digital gallery of the works created by the app with their timestamp, technique, and phrase used to generate it. This might also serve as a nice incentive for others to try out combinations that others have used.

**Comment by voodoohop** [comment]
My friend psilosamples is really into the Metaverse. He is constructing gallery spaces there. I took a little walk through it and it looked really cool. It's some open-source Metaverse platform. Will try and find out the name again.

@jacksongoode The *Galllery* page is on our to-do list for the next release. A live feed could also be interesting. At the moment the plan is to use social media as the database as we are trying to add as little backend logic as possible. (to keep it decentralized and light)
